### ISSUE TITLE
403 forbidden from jsdelivr when using noto icons

### ISSUE THREAD

**Issue by nberlette** [antfu/vscode-iconify#34]
### Describe the bug

The noto iconset is apparently too large for jsdelivr, and the extension is throwing a 403 forbidden error whenever I've used it in my code:

<img width="468" alt="Screen Shot 2022-09-18 at 3 44 47 PM" src="https://user-images.githubusercontent.com/11234104/190931347-e3c57612-4cb7-4224-af91-96f7f7cbdf66.png">

I intend on submitting a PR to fix this.

### Reproduction

Simply add an icon from the `noto` collection in your code, with the extension enabled. like `noto:t-rex`. Mouseover it and see what happens.

### System Info

```Shell
System:
    OS: Linux 5.15 Ubuntu 20.04.4 LTS (Focal Fossa)
    CPU: (16) x64 AMD EPYC 7B13
    Memory: 33.63 GB / 62.80 GB
    Container: Yes
    Shell: 5.0.17 - /usr/bin/bash
  Binaries:
    Node: 16.17.0 - ~/.nvm/versions/node/v16.17.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v16.17.0/bin/yarn
    npm: 8.15.0 - ~/.nvm/versions/node/v16.17.0/bin/npm

Also occurs on my personal machine, macOS 12.6 Monterey running on Apple Silicon
```

### Used Package Manager

n/a

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Amine27** [comment]
I am getting the same error. Can this PR been merged?

**Comment by ram-you** [comment]
A good solution for this is to make two entries in settings for  iconify icon-sets CDN : 
 - a main entry with jsdelivr  
 - a second entry with Github raw as a Fallback

So if the first fails we use the second automatically, a PR is needed for this.

**Comment by antfu** [comment]
Fixed in b86cff09b1dd10e646fc240ef8f9ffeabd93ee23
